js/fc.js

```diff
@@ -58,7 +58,7 @@ var FC = {
     MAX_SERVO_RATE: 125,
     MIN_SERVO_RATE: -125,
     isRatesInDps: function () {
```

Comment:
Why do we still have this function if it's always return true?

Reply:
Funny fact is that this fuction is already not neede at all :)